src/main/java/net/whistlingfish/harmony/protocol/MessageStartActivity.java

```diff
@@ -55,7 +56,12 @@ public IQ parseReplyContents(String statusCode, String errorString, String conte
             return OBJECT_MAPPER.convertValue(parseKeyValuePairs(statusCode, errorString, contents),
                     StartActivityReply.class);
         }
-
+        
+        @Override
+        public boolean validResponseCode(String code){
+        	//sometimes the start activity will return a 401 if a device is not setup correctly
+        	return super.validResponseCode(code) || code.equals("401");
+        }
```

Comment:
The use of the error codes seems to be erratic at logitech , apparently my apple tv is not BT paired with the hub, and so the last startActivity response gives a 401 "Bluetooth not paired" message, but otherwise everything is fine.